packages/features/CalendarEventBuilder.ts

```diff
@@ -226,12 +226,23 @@ export class CalendarEventBuilder {
 
     // Team & calendars
     if (eventType.team) {
+      // We need to get the team members assigned to the booking
+      // In the DB team members are stored in the Attendee table
+      const bookingAttendees = booking.attendees;
+
+      const hostsToInclude = eventType.hosts.filter((host) =>
+        bookingAttendees.some((attendee) => attendee.email === host.user.email)
+      );
+
       const hostsWithoutOrganizer = await Promise.all(
-        eventType.hosts.filter((h) => h.user.email !== user.email).map((h) => _buildPersonFromUser(h.user))
+        hostsToInclude
+          .filter((host) => host.user.email !== user.email)
+          .map((host) => _buildPersonFromUser(host.user))
       );
 
       const hostCalendars = [
-        ...eventType.hosts.map((h) => h.user.destinationCalendar).filter(Boolean),
+        user.destinationCalendar,
```

Comment:
Need to explicitly add the organizer's destination calendar since we are no longer looping through all event type hosts.